docs/ping-identity-oidc-setup.md

```diff
@@ -0,0 +1,50 @@
+# Ping Identity OIDC Authentication Provider Setup
+
+### Prerequisite: Configure Ping Identity Application
+
+Set up the Ping Identity environment and application. Refer to the [Ping Identity Application Setup](./ping-identity-env-setup.md) documentation for instructions.
+
+#### Edit Application Configuration
+
+Navigate to the `Application Overview > Configuration Tab` and click the edit button.
+
+- Check `Code` in `Response Type` and `Authorization Code` in `Grant Type`.
+
+![Response and Grant Type for Ping Identity Application](./images/response_and_grant_type_ping_identity_app.png){ width=30% }
+
+- Set Redirect URIs to `https://your-backstage.com/api/auth/oidc/handler/frame`.
+
+Navigate to the `Application Overview > Resources Tab` and click the edit button.
+
+- Add `email`, `offline_access` and `profile` to allowed scopes.
+
+![Allowed Scopes for Ping Identity Application](./images/allowed_scopes_ping_identity_app.png){ width=50% }
```

Comment:
Is there a way for `{ width=50% }` to not show up in the README.md? It looks like this right now, with  `{ width=50% }` mentioned next to the image:
![Screenshot from 2024-10-03 13-10-30](https://github.com/user-attachments/assets/a819abbc-6b19-463c-99a2-b66067a6a702)

Reply:
Good catch! I tried a bunch of different ways to get image resizing to work for both GitHub and TechDocs but there wasn't a good way to make it work for both (using html tag for images would not work because the image path is resolved differently). I just manually resized the image for now.